tasmota/my_user_config.h

```diff
@@ -637,6 +637,7 @@
   #define SDM120_SPEED         2400              // SDM120-Modbus RS485 serial speed (default: 2400 baud)
 //#define USE_SDM630                               // Add support for Eastron SDM630-Modbus energy monitor (+0k6 code)
   #define SDM630_SPEED         9600              // SDM630-Modbus RS485 serial speed (default: 9600 baud)
+//#define SDM630_IMPORT                          // Show import active energy in MQTT and Web (+0k3 code)
```

Comment:
Does this setting depend on ```USE_SDM630``` somehow?

Reply:
By default, Tasmota reads multiple ModBus registers from SDM630 but without 0x015A, 0X015C, 0x015E (phase 1, 2 and 3 import active energy). I have been using the extended version that reads these registers for four months now and it works very well. For users who need to read the consumed energy separately for each phase, there is this switch - #SDM630_IMPORT. This works ONLY if the SDM630 is compiled in the firmware as before (tasmota-sensors etc).

Reply:
Well, I mean, does the code compile if ```SDM630_IMPORT``` is defined but ```USE_SDM630``` is not?

Reply:
SDM630_IMPORT work only if USE_SDM630 is defined.